gravatar/src/main/java/com/gravatar/types/Hash.kt

```diff
@@ -0,0 +1,5 @@
+package com.gravatar.types
+
+public class Hash(private val hash: String) {
```

Comment:
Note: I made the hash param private to avoid code like: `email.hash().hash`, forcing `email.hash().toString()` seems better to me.